models/Translation.php

```diff
@@ -93,7 +93,7 @@ public static function IsAValidLanguage(string $domain, string $locale): bool
     }
 
     /**
-     * @return string|null
```

Comment:
@dvesh3 Should n't we keep this getter also as nullable ?

Reply:
good catch! changed it 😊